projects/WeTek.Play/options

```diff
@@ -0,0 +1,158 @@
+################################################################################
+# setup system defaults
+################################################################################
+
+  # The TARGET_CPU variable controls which processor should be targeted for
+  # generated code.
+    case $TARGET_ARCH in
+      arm)
+        # TARGET_CPU:
+        # arm2 arm250 arm3 arm6 arm60 arm600 arm610 arm620 arm7 arm7m arm7d
+        # arm7dm arm7di arm7dmi arm70 arm700 arm700i arm710 arm710c
+        # arm7100 arm720 arm7500 arm7500fe arm7tdmi arm7tdmi-s arm710t
+        # arm720t arm740t strongarm strongarm110 strongarm1100
+        # strongarm1110 arm8 arm810 arm9 arm9e arm920 arm920t arm922t
+        # arm946e-s arm966e-s arm968e-s arm926ej-s arm940t arm9tdmi
+        # arm10tdmi arm1020t arm1026ej-s arm10e arm1020e arm1022e
+        # arm1136j-s arm1136jf-s mpcore mpcorenovfp arm1156t2-s
+        # arm1176jz-s arm1176jzf-s cortex-a8 cortex-a9 cortex-r4
+        # cortex-r4f cortex-m3 cortex-m1 xscale iwmmxt iwmmxt2 ep9312.
+        #
+        TARGET_CPU="cortex-a9"
+
+        # TARGET_FLOAT:
+        # Specifies which floating-point ABI to use. Permissible values are:
+        # soft softfp hard
+        TARGET_FLOAT="hard"
+
+        # TARGET_FPU:
+        # This specifies what floating point hardware (or hardware emulation) is
+        # available on the target. Permissible names are:
+        # fpa fpe2 fpe3 maverick vfp vfpv3 vfpv3-fp16 vfpv3-d16 vfpv3-d16-fp16
+        # vfpv3xd vfpv3xd-fp16 neon neon-fp16 vfpv4 vfpv4-d16 fpv4-sp-d16
+        # neon-vfpv4.
+        TARGET_FPU="neon"
+        ;;
+    esac
+
+  # Bootloader to use (syslinux / u-boot / atv-bootloader / bcm2835-bootloader)
+    BOOTLOADER="u-boot"
+
+  # u-boot version to use (default)
+    UBOOT_VERSION=""
+
+  # Configuration for u-boot
+    UBOOT_CONFIG=""
+
+  # Target Configfile for u-boot
+    UBOOT_CONFIGFILE=""
+
+  # Kernel target for u-boot (default 'uImage' if BOOTLOADER=u-boot) (uImage / zImage)
+    KERNEL_UBOOT_TARGET="uImage-dtb"
+
+  # Kernel extra targets to build
+    KERNEL_UBOOT_EXTRA_TARGET="meson6_g18.dtd"
+
+  # Additional kernel make parameters (for example to specify the u-boot loadaddress)
+    KERNEL_MAKE_EXTRACMD=""
+
+  # Kernel to use. values can be:
+  # default:  default mainline kernel
+    LINUX="amlogic"
+
+
+################################################################################
+# setup build defaults
+################################################################################
+
+  # Build optimizations (size/normal)
+    OPTIMIZATIONS="size"
+
+  # Project CFLAGS
+    PROJECT_CFLAGS=""
+
+  # LTO (Link Time Optimization) support
+    LTO_SUPPORT="yes"
+
+  # GOLD (Google Linker) support
+    GOLD_SUPPORT="yes"
+
+  # SquashFS compression method (gzip / lzo / xz)
+    SQUASHFS_COMPRESSION="lzo"
```

Comment:
uhm. target image (SYSTEM) is ~ 10% smaller with gzip, no difference in boot times. with both lzo and gzip = ~ 13s +/- 100ms.

but looks like I am wrong. lzo is a bit faster at runtime.

with gzip:
```
Wetek:~ # echo 3 > /proc/sys/vm/drop_caches; dd if=/usr/lib/kodi/kodi.bin of=/dev/null
...
18597832 bytes (17.7MB) copied, 0.634658 seconds, 27.9MB/s
18597832 bytes (17.7MB) copied, 0.720829 seconds, 24.6MB/s
18597832 bytes (17.7MB) copied, 0.687909 seconds, 25.8MB/s
18597832 bytes (17.7MB) copied, 0.784210 seconds, 22.6MB/s
```

with lzo
```
...
18597832 bytes (17.7MB) copied, 0.542292 seconds, 32.7MB/s
18597832 bytes (17.7MB) copied, 0.549667 seconds, 32.3MB/s
18597832 bytes (17.7MB) copied, 0.583566 seconds, 30.4MB/s
18597832 bytes (17.7MB) copied, 0.558868 seconds, 31.7MB/s
18597832 bytes (17.7MB) copied, 0.617791 seconds, 28.7MB/s
18597832 bytes (17.7MB) copied, 0.594130 seconds, 29.9MB/s
```

so better keep lzo. I swear I got different results one year back when I last tested on x86. I must re-test that again.. ;)

sorry for the noise.